src/Equinox.MessageDb/MessageDb.fs

```diff
@@ -311,8 +320,27 @@ type MessageDbContext(connection : MessageDbConnection, batchOptions : BatchOpti
         let! version, events = Read.loadForwardsFrom log connection.ReadRetryPolicy connection.Reader batchOptions.BatchSize batchOptions.MaxBatches streamName 0L requireLeader
         return Token.create version, Array.chooseV tryDecode events }
     member _.LoadLast(streamName, requireLeader, log, tryDecode) : Async<StreamToken * 'event array> = async {
-        let! version, events = Read.loadLastEvent log connection.ReadRetryPolicy connection.Reader requireLeader streamName
+        let! version, events = Read.loadLastEvent log connection.ReadRetryPolicy connection.Reader requireLeader streamName None
         return Token.create version, Array.chooseV tryDecode events }
+    member _.LoadSnapshot(category, streamId, requireLeader, log, tryDecode, eventType) = async {
+        let snapshotStream = StreamName.create (category + ":snapshot") streamId |> StreamName.toString
```

Comment:
For better or worse, equinox.core.streamname.render is used for this in most other places